bandit/core/context.py

```diff
@@ -205,19 +202,9 @@ def _get_literal_value(self, literal):
         elif isinstance(literal, ast.Dict):
             literal_value = dict(zip(literal.keys, literal.values))
 
-        elif isinstance(literal, ast.Ellipsis):
-            # what do we want to do with this?
-            literal_value = None
```

Comment:
Is this necessary to remove?

Reply:
% `uvx python3.14 -c"import ast ; ast.Ellipsis"`
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
    import ast ; ast.Ellipsis
                 ^^^^^^^^^^^^
AttributeError: module 'ast' has no attribute 'Ellipsis'
```